gwt-material/src/main/java/gwt/material/design/client/ui/MaterialCutOut.java

```diff
@@ -0,0 +1,297 @@
+package gwt.material.design.client.ui;
+
+/*
+ * #%L
+ * GwtMaterial
+ * %%
+ * Copyright (C) 2015 GwtMaterialDesign
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.google.gwt.dom.client.Document;
+import com.google.gwt.dom.client.Element;
+import com.google.gwt.dom.client.Style;
+import com.google.gwt.dom.client.Style.Display;
+import com.google.gwt.dom.client.Style.Overflow;
+import com.google.gwt.dom.client.Style.Position;
+import com.google.gwt.dom.client.Style.Unit;
+import com.google.gwt.event.logical.shared.CloseEvent;
+import com.google.gwt.event.logical.shared.CloseHandler;
+import com.google.gwt.event.logical.shared.HasCloseHandlers;
+import com.google.gwt.event.logical.shared.ResizeEvent;
+import com.google.gwt.event.logical.shared.ResizeHandler;
+import com.google.gwt.event.shared.HandlerRegistration;
+import com.google.gwt.user.client.DOM;
+import com.google.gwt.user.client.Window;
+import com.google.gwt.user.client.Window.ScrollEvent;
+import com.google.gwt.user.client.Window.ScrollHandler;
+import com.google.gwt.user.client.ui.HTMLPanel;
+import com.google.gwt.user.client.ui.RootPanel;
+
+//@formatter:off
+/**
+ * MaterialCutOut is a fullscreen modal-like component to show users about new
+ * features or important elements of the document.
+ * 
+ * You can use {@link CloseHandler}s to be notified when the cut out is closed.
+ * 
+ * <h3>UiBinder Usage:</h3>
+ * 
+ * <pre>
+ * {@code
+ * <m:MaterialCutOut ui:field="cutOut">
+ *      <!-- add any components or html elements here -->
+ * </m:MaterialCutOut>
+ * }
+ * </pre>
+ * 
+ * <h3>Java Usage:</h3>
+ * {@code
+ * MaterialCutOut cutOut = ... //create using new or using UiBinder
+ * cutOut.setTarget(myTargetElement); //the element you want to focus
+ * cutOut.openCutOut(); //shows the modal over the page
+ * }
+ * 
+ * <h3>Styling:</h3> You use change the cut out style by using the
+ * <code>material-cutout</code> class, and <code>material-cutout-focus</code>
+ * class for the focus box.
+ * 
+ * @author gilberto-torrezan
+ *
+ */
+// @formatter:on
+public class MaterialCutOut extends HTMLPanel implements HasCloseHandlers<MaterialCutOut> {
+
+    private String backgroundColor = "rgba(30,136,229,0.85)";
```

Comment:
As commented above please use the existing setBackgroundColor of a  ComplexWidget, also for the opacity lets put it like a final property or you can do bakckgroundOpacity or something. Its indicated on Material Design aspecs that we will have a uniform palette of colors. :)